requirements.txt

```diff
@@ -18,3 +18,5 @@ pyyaml
 Pillow
 requests
 python-dateutil
+pytest
```

Comment:
I removed these deliberately recentlyhere because we changed the structure of the test workflows 

https://github.com/ContactEngineering/SurfaceTopography/commit/1ab47cf01f62b1ad55b9b2194d22ea0ee0a3a6ec

Reply:
please remove this again